internal/manifests/collector/ports.go

```diff
@@ -145,11 +157,19 @@ func getReceiverServicePort(logger logr.Logger, serviceAddress string, receiverN
 			if _, ok := servicePortsMap[port]; ok {
 				logger.Info("Duplicate port has been configured in Agent Config for port", zap.Int32("port", port))
 			} else {
+				var serviceName string
+				if len(receiverName) < 12 {
+					serviceName = CWA + receiverName
```

Comment:
Why specifically 12 ?

Reply:
We have a limit set to for length to be <15 and since len(CWA)=3, 12+3 would be 15.

Reply:
And lisa I changed the reciever names to appsig-*. We already have truncating but it would truncate till cwa-appsignals- which ends in a hyphen, so I just changed the names.